linode/manager
# [DI-18311] - Adding CloudPulse section to Cloud Manager

## Description 📝
Added a CloudPulseTab component 
Added a dashboard landing component
Added a RegionSelect component
Added a TimeRange component
Added a Global Filters component to manage the filters

## Changes  🔄
List any change relevant to the reviewer.
- Added a CloudPulseTab component 
- Added a dashboard landing component
- Added a RegionSelect component
- Added a TimeRange component
- Added a Global Filters component to manage the filters

## Preview 📷
**Include a screenshot or screen recording of the change**

:bulb: Use `<video src="" />` tag when including recordings in table.

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-04-22 at 19 55 04](https://github.com/linode/manager/assets/159890961/994de78b-2e8a-4bc3-a593-1d08165c6f95) | ![Screenshot 2024-04-23 at 17 52 03](https://github.com/linode/manager/assets/159890961/a84f965b-1109-4808-b0fd-59acfb3cd283) |
| ![Screenshot 2024-04-22 at 19 55 04](https://github.com/linode/manager/assets/159890961/994de78b-2e8a-4bc3-a593-1d08165c6f95) | ![Screenshot 2024-04-23 at 17 51 51](https://github.com/linode/manager/assets/159890961/4731bb0a-d36f-40bc-9532-51a65c711444) |


## How to test 🧪

### Prerequisites
(How to setup test environment)
- Follow cloud manager standard procedures

### Reproduction steps
(How to reproduce the issue, if applicable)
- N/A

### Verification steps
(How to verify changes)
- Navigate to cloud manager and once feature flag of CloudPulse is enabled then navigate to the Monitor section in the sidebar for the CloudPulse section.
- There the added two global filters options are visible, can be used to select the respective options for both the RegionSelect and TimeRange 

## As an Author I have considered 🤔

*Check all that apply*
